index.js

```diff
@@ -0,0 +1,48 @@
+const books = document.getElementById('books');
+
+const myLibrary = [];
+
+function Book(author, title, pages) {
+  this.author = author;
+  this.title = title;
+  this.pages = pages;
+}
+
+function addBookToLibrary(e) {
+  e.preventDefault();
+  const li = document.createElement('li');
+
+  li.className = 'collection-item';
+  const [title, author, pages] = e.target.elements;
+  const book = new Book(title.value, author.value, pages.value);
+
+  li.innerHTML = `
+  <h5>Author: <span id="author">${book.author}</span></h5>
+  <p>Title: ${book.title} </p>
+  <p>Pages: ${book.pages}</span></p>
+  <a class="delete"><i class="far fa-trash-alt"></i></a>
+  <label>
+  <input type="checkbox" />
+  <span>Read</span>
+</label>
+  `;
```

Comment:
Great work using this method to display your books 👏🏿 . However, it's a hard requirement to use a loop to display your book items in the array. Kindly check below ⬇️ 

- [ ] Kindly create a `Display Book` method that loops through the array and displays the items in the array. This is a hard requirement for this project and hence has to be implemented before your project can be approved ⬇️ 

![books_render_loop](https://user-images.githubusercontent.com/32361963/115721430-b3a7dd00-a386-11eb-9682-240fda19c79e.png)